### ISSUE TITLE
Potatoes Do Not Survive the Year

### ISSUE THREAD

**Issue by bedalton** [Creatures-Developer-Network/C2toDS-Refresh#51]
So the potato plants do not survive their first year

Watching the potatoes, I have noticed in three worlds, that they migrate left over time where they eventually end up in the waterwheel area.

The second problem is that they only live a few minutes. They die and regrow many times a day. I think they should have a lifespan of at least a day, but I do not know how much of this is in line with the original C2.

The rapid life/death cycle, coupled with the migration left results in the potatoes not surviving through winter.

**Comment by Amaikokonut** [comment]
That's really funny that they're "migrating;" have you noticed when/how that occurs? Like are they just always spitting their seeds left or something or are they physically moving that direction for some reason?

**Comment by bedalton** [comment]
There is a seed script, but I never see seeds. The potatoes just grow and die and then another pops up. Every so often one shows up to the right, but then it dies and comes back to the left typically.

I'll need to look more into the potatoes and see how they grow.

Any thoughts on how long a potato should live. Less than two minutes game time I do not think would even let a creature eat the potato. They are only "ripe" for just over a minute.

**Comment by Amaikokonut** [comment]
Yeah, to me it makes sense to have them at least live out the year. Real-world potatoes have a long shelf life when stored properly after all!

Have you noticed any similar issues with the carrots? I seem to remember the two using very similar scripts but maybe I'm wrong.

I wonder if migration is a weird issue of the scripts using a mix of `posl/post` and `posx/posy` without consistency. That's just a wild guess but it's something I've seen in the past that can cause unintended 'migration'
